qemu/target-i386/int_helper.c

```diff
@@ -352,7 +352,7 @@ static int idiv64(uint64_t *plow, uint64_t *phigh, int64_t b)
         neg128(plow, phigh);
     }
     sb = (b < 0);
-    if (sb) {
+    if (sb && (b != 0x8000000000000000LL)) {
```

Comment:
this looks like a trick here. sorry but i feel it is ugly to fix a bug this way. any ideas why oss-fuzzer reports this issue?

Reply:
The min int8_t is -128(0x80), the max is 127(0x7f), the abs value of min is bigger than max and overflowed. In most machine and compiler, -(-128) = ~(-128) + 1 = -128. Same ruler for other int types. I think the oss-fuzz just report this bug is not good compatible, just like left shift the negative int. This code change keeps the original logic and becomes compatible.